.vscode/tasks.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "build agent",
+            "command": "invoke agent.build",
```

Comment:
How does all of those work with local python venv ?

Reply:
I think as long as `tasks` is a template file (`tasks.json.template`), `invoke agent.build` is a sane default.

Reply:
Yeah I am leaning to making everything a template so people can change it more freely.
It seems like there are a lot of possible config/environment options and it would be difficult to capture all of them.